test/reverseRegistrar/TestL2ReverseRegistrar.js

```diff
@@ -241,6 +287,57 @@ describe('L2ReverseRegistrar', function () {
         ),
       ).to.be.revertedWith(`InvalidSignature()`)
     })
+
+    it('reverts if inceptionDate is too low', async () => {
```

Comment:
a bit pedantic but this test says `inceptionDate` while the other test for `setNameForAddrWithSignature ` says `inception date`. Probably better to standardise to either